main/params.go

```diff
@@ -66,252 +67,262 @@ var (
 	errInvalidStakerWeights = errors.New("staking weights must be positive")
 )
 
-// Parse the CLI arguments
-func init() {
-	loggingConfig, err := logging.DefaultConfig()
-	if err != nil {
-		Err = err
-		return
-	}
-
+// avalancheFlagSet returns the complete set of flags for avalanchego
+func avalancheFlagSet() *flag.FlagSet {
 	fs := flag.NewFlagSet(constants.AppName, flag.ContinueOnError)
 
 	// If this is true, print the version and quit.
-	version := fs.Bool("version", false, "If true, print version and quit")
+	fs.Bool(versionKey, false, "If true, print version and quit")
```

Comment:
We should use consts for both the flag names (e.g. `versionKey` on this line) and the default value (e.g. `false` should be replaced with a const `defaultVersionVal`)

We can define the keys and values next to each other for clarity.

e.g.

```
const (
  versionKey = "version"
  defaultVersionVal = false
)
```

Reply:
This seems to make it less clear and it adds a variable that will only ever be used once, so going to leave it as is.